Update features in neo4jgraphql class docs (#281)
neo4j/docs-graphql: modules/ROOT/pages/neo4jgraphql-class.adoc

        aggregationFilters?: boolean;
        nestedUpdateOperationsFields?: boolean;
    };
    unsafeEscapeOptions?: {
        /** Disables automatic escaping of node labels.
         *
         * **WARNING**: Disabling label escaping may lead to code injection and unsafe Cypher.
         */
        disableNodeLabelEscaping?: boolean;
        /** Disables automatic escaping of relationship types.
         *
         * **WARNING**: Disabling type escaping may lead to code injection and unsafe Cypher.
         */
        disableRelationshipTypeEscaping?: boolean;
    };
    vector?: Neo4jVectorSettings;
    cypherDirective?: {

>         /** Enabling this setting, the limit generated by the directive `@limit` won't be added to fields with custom cypher */

Reviewer: ```suggestion
        /** Enable this setting to prevent the limit generated by the directive `@limit` from being added to fields with custom Cypher. */
```
